### PR TITLE
RRS-120 Policy Documents

### PR DESCRIPTION
**Pull request by OttoChu**
I don't know what happened to the last merge. I just recreated it so it is fine

### CODE REVIEW

File: app/privacy-policy/page.tsx [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+      <br />
+      <h2 className='text-2xl font-semibold mb-1 w-fit text-accent'>
+        5. Your Choices
+      </h2>
+      <li className='ml-10 w-fit'>
+        <span className='text-accent'>
+          Account Information
+        </span>
+        <p>
+          You can update or delete your account information at any time by accessing your
```
Comment (LukeHosk) [inline app/privacy-policy/page.tsx]: Can you delete the account in settings atm?

Maybe provide a contact email?

File: app/privacy-policy/page.tsx [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+          account settings.
+        </p>
+      </li>
+
+      <li className='ml-10 w-fit'>
+        <span className='text-accent'>
+          Cookies
```
Comment (LukeHosk) [inline app/privacy-policy/page.tsx]: We probably need a cookie banner?

We currently only* use cookies for auth so maybe a disclaimer on the login page is enough?

*: technically also used for feature flag overrides, but you can only use those as an admin so typical users will never encounter

File: app/privacy-policy/page.tsx [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+      <p className='ml-10'>
+        If you have any questions, concerns, or feedback regarding this Privacy Policy or our
+        privacy practices, please contact us at
+        {' '}
+        <a
+          className='underline text-blue-500 transition-colors duration-300 ease-in-out
+          hover:text-blue-600'
+          href='mailto:help@rentalreview.com'
+        >
+          help@rentalreview.com
```
Comment (LukeHosk) [inline app/privacy-policy/page.tsx]: Does this email exist?

File: app/privacy-policy/page.tsx [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+        </a>
+        .
+      </p>
+
+      <i className='flex-1 flex flex-col justify-center items-center py-20 text-accent'>
+        Note: This document is generated by ChatGPT.
```
Comment (LukeHosk) [inline app/privacy-policy/page.tsx]: Legal document generated by chat gpt feels like a bad idea?

File: app/terms-of-service/page.tsx [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+      </li>
+      <li className='ml-10 w-fit'>
+        You may not engage in any conduct that violates the rights of others, including but
+        not limited to infringing intellectual property rights, harassing or defaming others,
+        or distributing spam or malware.
+      </li>
+
+      <br />
+      <h2 className='text-2xl font-semibold mb-1 w-fit text-accent'>
+        4. Reviews and Ratings
```
Comment (LukeHosk) [inline app/terms-of-service/page.tsx:59]: If we're going to allow image uploads, we probably want a notice about only allowing relevant and appropriate images to be uploaded. also point to our moderation system

**Review by LukeHosk — CHANGES_REQUESTED** [review]

**Comment by OttoChu** [comment]
The new email exists and the login details are located in the file so if anyone in our team can login to the account

**Review by LukeHosk — APPROVED** [review]

**Pull request merged**